### PR TITLE
fromString: add option to parse string with lat first

### PR DESCRIPTION
**Pull request by evansiroky**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -129,17 +129,21 @@ module.exports.fromPoint = fromPoint
  * Tries to parse from a string.
  *
  * @memberof conveyal/lonlat
- * @param  {string} str A string in the format: `longitude,latitude`
- * @return {lonlat}
- * @throws {InvalidCoordinateException}
+ * @param  {string} str                 A string in the format: `longitude,latitude`
+ * @param  {boolean} [latIsFrist=false] Whether or not the first value is latitude.
```
Comment (landonreed) [inline index.js]: @evansiroky, I found a sneaky typo `latIsFrist`.  Might want to do a replaceAll

File: index.test.js [hunk 2]
```diff
@@ -93,6 +94,9 @@ describe('lonlat', () => {
     }, {
       calculated: ll.fromString(str),
       description: 'String with comma separating lon and lat'
```
Comment (landonreed) [inline index.test.js]: Might be worth adding `, respectively` to this description and the one below for even more clarity.

**Review by landonreed — CHANGES_REQUESTED** [review]
Just a couple of minor things.

**Review by landonreed — APPROVED** [review]

**Comment by trevorgerhardt** [comment]
@evansiroky & @landonreed we should think more about this implementation. If we *know* lat is first then I would say we should consider it a new form or structure. This library is meant to be as explicit as possible (hence all the functions already). I would rather not add options to the conversion functions and rather explicitly create a new function called `fromLatFirstString` to be used in this instance.

**Comment by trevorgerhardt** [comment]
`fromLatLonString` would also work.

**Comment by trevorgerhardt** [comment]
But `fromLatFirstString` is more explicit

**Comment by evansiroky** [comment]
See #14

**Pull request merged**